layers/+tools/sailfish-scratchbox/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- sailfish scratchbox layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2017 Victor Polevoy
+;;
+;; Author: Victor Polevoy <fx@thefx.co>
+;; URL: https://github.com/vityafx/sailfish-scratchbox-layer
```

Comment:
URL should point be the URL of Spacemacs. Besides, the current URL gives me a 404 error - is it a private repo?

Reply:
Ehm, it existed before I decided to move it into the spacemacs. I have deleted it already. Gonna change to URL of Spacemacs.